### ISSUE TITLE
Mathematical expression support for markdown

### ISSUE THREAD

**Issue by wackerow** [ethereum/ethereum-org-website#7653]
**Is your feature request related to a problem? Please describe.**
Currently our markdown does not support syntax for mathematical expressions. Although these are not super frequent, having the ability to properly display mathematical expressions could expand our ability to convey certain concepts. I suspect we could use more math examples if we had the proper support for them. 

Surfaced while working through the new page related to proof-of-stake rewards and penalties. 

**Describe the solution you'd like**
https://github.blog/2022-05-19-math-support-in-markdown/
GitHub recently rolled out a similar feature, which can be used with `$` or `$$` delimiters. ie. $e^{i \pi} = -1$ instead of `e^(i*pi) = -1`
Not sure exactly what library they use.
Couple possible libraries
https://www.gatsbyjs.com/plugins/gatsby-remark-katex/
https://www.mathjax.org/

**Describe alternatives you've considered**
Just leave the markdown as is, and type all math examples with inline text. :-(

**Additional context**
The method chosen should be sure to support dark mode color themes.
Ideally text is still selectable, rather than rendered as an image. 
Ideally also support RTL languages, but not certain how that would play in.

Example of mathematic expressions being used: https://eth2book.info/altair/annotated-spec/#reward-and-penalty-calculations

**Want to contribute?**
We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

**Comment by samajammin** [comment]
I believe KaTeX is a fairly popular one. @nhsz may have insight, he implemented math/latex support on https://github.com/ethereum/cryptography-research-website

**Comment by kumarkolhe** [comment]
I used mathjax using the following reference:

[https://math.meta.stackexchange.com/questions/5020/mathjax-basic-tutorial-and-quick-reference](url)

Then I tried to copy the mathematical expression from [https://eth2book.info/altair/annotated-spec#reward-and-penalty-calculations](url) 
![mathjsx](https://user-images.githubusercontent.com/62575665/215288426-3fff68f8-d10c-4ebd-b51c-7fd9b61c3465.PNG)

Here is my forked repo link:
[https://github.com/kumarkolhe/ethereum-org-website/blob/dev/src/content/reward-and-penalty-calculations/index.md](url)

**Comment by wackerow** [comment]
Closing this out for now with concern for translation process... if more interest is expressed can resurface in the future.
